languages/js/src/Polar.ts

```diff
@@ -208,23 +214,35 @@ export class Polar {
       ffiQuery = this.#ffiPolar.newQueryFromTerm(term);
     }
     this.processMessages();
-    return new Query(ffiQuery, host).results;
+    return new Query(ffiQuery, host, bindings).results;
   }
 
   /**
    * Query for a Polar rule.
    */
-  queryRule(name: string, ...args: unknown[]): QueryResult {
-    return this.query(new Predicate(name, args));
+  queryRule(name: string, bindings?: any, ...args: unknown[]): QueryResult {
+    return this.query(new Predicate(name, args), bindings);
```

Comment:
this is a little confusing coming from the other host libraries. maybe a separate `queryRuleWithBindings` function?

Reply:
Yeah I like that better